proposals/0005-DistributedTracing.md

```diff
@@ -0,0 +1,84 @@
+# Distributed tracing
+
+* Proposal: [MP-0005](0005-DistributedTracing.md)

```

Comment:
A proposal with ID `MP-0005` already exists (JWT) - could we change this to `MP-0007` to avoid conflicts with #18 and #23 please?